event-sauce/src/sync.rs

```diff
@@ -0,0 +1,67 @@
+use crate::{Event, EventData, UpdateEventBuilder};
+use serde::{Deserialize, Serialize};
+
+/// ConflictEvent definition
+///
+/// The ConflictEvent is created when there are conflicting modifications
+/// done by distinct actors detected. It refers to the Event that has already
+/// been applied and the EventData of would-be Event, which could not be
+/// applied due to the conflict.
+//#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct SyncConflict<EDA, EDC>
+where
+    EDA: EventData, // Applied Event<EDA>
+    EDC: EventData, // Conflicting would-be Event<EDC> that could not be applied due to the conflict
```

Comment:
Move these comments into the doc block so they show up in the rendered docs.

```suggestion
    EDA: EventData,
    EDC: EventData,
```